AlfrescoKit/AlfrescoKit/Picking View Controller/AKAlfrescoNodePickingListViewController.m

```diff
@@ -0,0 +1,258 @@
+                                              delegate:(id<AKAlfrescoNodePickingListViewControllerDelegate>)delegate
+                                               session:(id<AlfrescoSession>)session
+{
+    return [self initAlfrescoFolderPickerWithRootFolder:folder selectedNodes:selectedNodes delegate:delegate listingContext:nil session:session];
+}
+
+- (instancetype)initAlfrescoFolderPickerWithRootFolder:(AlfrescoFolder *)folder
+                                         selectedNodes:(NSMutableArray *)selectedNodes
+                                              delegate:(id<AKAlfrescoNodePickingListViewControllerDelegate>)delegate
+                                        listingContext:(AlfrescoListingContext *)listingContext
+                                               session:(id<AlfrescoSession>)session
+{
+    self = [self initWithAlfrescoFolder:folder listingContext:listingContext session:session];
+    if (self)
+    {
+        self.nodePickerOptions = AKAlfrescoNodePickerTypeFolderPicker;
+        self.delegate = delegate;
+        self.selectedNodes = (selectedNodes) ? selectedNodes : [NSMutableArray array];
+    }
+    return self;
+}
+
+- (instancetype)initAlfrescoDocumentPickerWithRootFolder:(AlfrescoFolder *)folder
+                                       multipleSelection:(BOOL)allowMultiple
+                                           selectedNodes:(NSMutableArray *)selectedNodes
+                                                delegate:(id<AKAlfrescoNodePickingListViewControllerDelegate>)delegate
+                                                 session:(id<AlfrescoSession>)session
+{
+    return [self initAlfrescoDocumentPickerWithRootFolder:folder multipleSelection:allowMultiple selectedNodes:selectedNodes delegate:delegate listingContext:nil session:session];
+}
+
+- (instancetype)initAlfrescoDocumentPickerWithRootFolder:(AlfrescoFolder *)folder
+                                       multipleSelection:(BOOL)allowMultiple
+                                           selectedNodes:(NSMutableArray *)selectedNodes
+                                                delegate:(id<AKAlfrescoNodePickingListViewControllerDelegate>)delegate
+                                          listingContext:(AlfrescoListingContext *)listingContext
+                                                 session:(id<AlfrescoSession>)session
+{
+    self = [self initWithAlfrescoFolder:folder listingContext:listingContext session:session];
+    if (self)
+    {
+        self.nodePickerOptions = AKAlfrescoNodePickerTypeDocumentPicker;
+        if (allowMultiple)
+        {
+            self.nodePickerOptions |= AKAlfrescoNodePickerTypeMultiplePicker;
+        }
+        self.delegate = delegate;
+        self.selectedNodes = (selectedNodes) ? selectedNodes : [NSMutableArray array];
+        
+    }
+    return self;
+}
+
+- (NSString *)nibName
+{
+    return NSStringFromClass([self superclass]);
```

Comment:
Not sure I follow?

The only reason to subclass this picking controller would be to customize the cells. This would use the parent's view, tableview etc. and only update the cells as overridden by cellForIndexPath:. The controller should still look/behave the same.